Gemfile.lock

```diff
@@ -3,7 +3,7 @@ GEM
   specs:
     RedCloth (4.2.9)
     blankslate (2.1.2.4)
-    celluloid (0.16.0)
+    celluloid (0.16.1)
```

Comment:
Doesn't this undo https://github.com/artsy/artsy.github.io/commit/88e69eb38dd12f69972a08664bac89a97e47f59e ? [Version 0.16.1](https://rubygems.org/gems/celluloid/versions/0.16.1) is still yanked.